src/OrbitQt/orbitmainwindow.cpp

```diff
@@ -470,7 +470,7 @@ void OrbitMainWindow::SetupCaptureToolbar() {
   // Timer
   toolbar->addWidget(CreateSpacer(toolbar));
   QFontMetrics fm(ui->timerLabel->font());
-  int pixel_width = fm.width("w");
+  int pixel_width = fm.horizontalAdvance("w");
```

Comment:
This kind of comes out of nothing. Is this part of "minor bugfixes"?

Reply:
width is deprecated so I replaced it. I do not think this fixes anything. just code cleanup.